scripts/lib/upgrade-zulip-stage-2

```diff
@@ -57,6 +58,12 @@ try:
 except (configparser.NoSectionError, configparser.NoOptionError):
     tornado_processes = 1
 
+if glob.glob("/usr/share/postgresql/*/extension/tsearch_extras.control"):
+    # Remove legacy tsearch_extras package references
+    subprocess.check_call(["su", "postgres", "-c",
+                           'psql zulip -c "DROP EXTENSION IF EXISTS tsearch_extras;"'])
```

Comment:
Should we stick with our usual convention of `psql -v ON_ERROR_STOP=1`? The `IF EXISTS` handles the case where the extension is missing. If something weirder is going on, we probably don’t want to continue with removing the package because that could make the problem worse.